### PR TITLE
tab indents fixed

### PR DESCRIPTION
**Pull request by nefertarii86**

### CODE REVIEW

File: To_Do_List.html [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
-                                 <button type="button" class="btn btn-success">FINISHED</button> </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-        <script src="./js/bootstrap.bundle.min.js"> </script>
+      <div class="container border p-5 my-5"> 
+        <div class="text-dark text-center">
+          <h5>To Do App</h5>
+            </div>
```
Comment (computeus) [inline To_Do_List.html]: fix here @nefertarii86

File: To_Do_List.html [hunk 2]
```diff
@@ -1,68 +1,68 @@
...
+                </tr>
+                <tr>
+                  <th scope="row">3</th>
+                    <td>Sign up for online course</td>
+                    <td>In progress</td>
+                    <td><button type="button" class="btn btn-danger">DELETE</button>
+                        <button type="button" class="btn btn-success">FINISHED</button> </td>
+                </tr>
+              </tbody>
+              </table>
```
Comment (computeus) [inline To_Do_List.html]: fix here @nefertarii86

File: To_Do_List.html [hunk 3]
```diff
@@ -1,68 +1,68 @@
...
+                    <td>In progress</td>
+                    <td><button type="button" class="btn btn-danger">DELETE</button>
+                        <button type="button" class="btn btn-success">FINISHED</button>
+                    </td>
+                </tr>
+                <tr>
+                  <th scope="row">3</th>
+                    <td>Sign up for online course</td>
+                    <td>In progress</td>
+                    <td><button type="button" class="btn btn-danger">DELETE</button>
```
Comment (computeus) [inline To_Do_List.html]: one button per line

File: To_Do_List.html [hunk 4]
```diff
@@ -1,68 +1,68 @@
 <!DOCTYPE html>
-<html lang="en">
+  <html lang="en">
```
Comment (computeus) [inline To_Do_List.html]: don't indent here

Comment (nefertarii86) [reply To_Do_List.html]: Okey!

File: To_Do_List.html (hunk 1 above)
Comment (nefertarii86) [reply To_Do_List.html]: Done.

File: To_Do_List.html (hunk 2 above)
Comment (nefertarii86) [reply To_Do_List.html]: Done

File: To_Do_List.html (hunk 3 above)
Comment (nefertarii86) [reply To_Do_List.html]: Okey

File: To_Do_List.html [hunk 5]
```diff
@@ -1,68 +1,72 @@
...
+          <tr>
+            <th scope="col"> No. </th>
+            <th scope="col"> Todo item </th>
+            <th scope="col"> Status </th>
+            <th scope="col"> Action </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">1</th>
```
Comment (computeus) [inline To_Do_List.html:41]: fix here @nefertarii86

File: To_Do_List.html [hunk 6]
```diff
@@ -1,68 +1,72 @@
...
+              <td>Buy groceries for next week</td>
+              <td>In progress</td>
+              <td>
+                <button type="button" class="btn btn-danger">DELETE</button> 
+                <button type="button" class="btn btn-success">FINISHED</button>
+              </td>
+          </tr>
+          <tr>
+            <th scope="row">2</th>
```
Comment (computeus) [inline To_Do_List.html:50]: fix here @nefertarii86

File: To_Do_List.html [hunk 7]
```diff
@@ -1,68 +1,72 @@
...
+              <td>Renew car insurance</td>
+              <td>In progress</td>
+              <td>
+                <button type="button" class="btn btn-danger">DELETE</button>
+                <button type="button" class="btn btn-success">FINISHED</button>
+              </td>
+          </tr>
+          <tr>
+            <th scope="row">3</th>
+              <td>Sign up for online course</td>
```
Comment (computeus) [inline To_Do_List.html]: fix here @nefertarii86

**Comment by computeus** [comment]
Fixes #5

**Pull request merged**